db/usage/id_digitizer.m4

```diff
@@ -97,7 +98,9 @@ ID(`0066', `utf16le_char_gesture_enc',    `Sel',  `utf16 little endian character
 ID(`0067', `utf16be_char_gesture_enc',    `Sel',  `utf16 big endian character gesture encoding')dnl
 ID(`0068', `utf32le_char_gesture_enc',    `Sel',  `utf32 little endian character gesture encoding')dnl
 ID(`0069', `utf32be_char_gesture_enc',    `Sel',  `utf32 big endian character gesture encoding')dnl
-ID(`006A', `gesture_char_enable',         `DF',   `gesture character enable')dnl
+ID(`006A', `gesture_char_enable',         `DF',   `gesture character enable')dnl  NOTE: Currently in conflict with HUTRR87 "Capacitive Heat Map Protocol Vendor ID"
```

Comment:
Could you put the comment *above* the line? It's getting too long otherwise.

Reply:
Fixed